public/index.html

```diff
@@ -15,6 +15,11 @@
     <meta name="twitter:title" content="$OG_TITLE" />
     <meta name="twitter:description" content="$OG_DESCRIPTION" />
     <meta name="twitter:image" content="$OG_IMAGE" />
+    <script
+      async
+      src="https://platform.twitter.com/widgets.js"
+      charset="utf-8"
+    ></script>
```

Comment:
https://dev.to/heymarkkop/embed-twitter-widget-on-reactjs-1768
you can probably use this package https://www.npmjs.com/package/react-twitter-embed

Reply:
I was trying to avoid using a package, but I can use it if you like